src/host/getset.cpp

```diff
@@ -1516,6 +1626,16 @@ void ApiRoutines::GetConsoleDisplayModeImpl(ULONG& flags) noexcept
     LockConsole();
     auto Unlock = wil::scope_exit([&] { UnlockConsole(); });
 
-    ServiceLocator::LocateGlobals().getConsoleInformation().SetTitle(title);
+    auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
+    gci.SetTitle(title);
+
+    if (auto writer = gci.GetVtWriter())
+    {
+        writer.WriteUTF8("\x1b]0;");
+        writer.WriteUTF16StripControlChars(title);
+        writer.WriteUTF8("\x7");

```

Comment:
true ST would go here if we used it


```suggestion
        writer.WriteUTF8("\x1b\\");
```